android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt

```diff
@@ -14,263 +14,256 @@ import java.util.UUID
 @ReactModule(name = PlayerModule.name)
 class PlayerModule(private val context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
     /**
-     * In-memory mapping of `id` <-> `Player`.
+     * In-memory mapping of `nativeId` strings and `Player` instances.
      */
     private var registry: MutableMap<String, Player> = mutableMapOf()
```

Comment:
Alright, that'd be better. I'll use an inline class for it.

Reply:
I ended up using type aliases because they were both easier to implement and replicate in Swift too: fd8ef12d29882cff5479c4f13aa8064100555ab7

Is this what you had in mind?

Reply:
Nice improvement 👍